cmd/sops/common/common.go

```diff
@@ -12,6 +12,7 @@ import (
 	"go.mozilla.org/sops/keyservice"
 	"go.mozilla.org/sops/stores/json"
 	"go.mozilla.org/sops/stores/yaml"
+	"go.mozilla.org/sops/stores/env"
```

Comment:
I don't think we should call this "env". It's too generic and confusing compared to regular env variables (which aren't files).

Reply:
Agreed. Maybe 'dotenv' would be a better name.

Reply:
I like "dotenv". Will change.